packages/config/src/case.js

```diff
@@ -27,7 +25,7 @@ const normalizeBuildCase = function ({
   publish = Publish,
   ...build
 } = {}) {
-  return removeFalsy({
+  return {
```

Comment:
This is now performed later, as part of `normalize.js`. This allows `priorityConfig` (config changes from plugins) to override properties with empty strings.